lua/harpoon/ui.lua

```diff
@@ -47,16 +51,13 @@ local function get_menu_items()
     return indices
 end
 
-local save_changes = function()
-    Marked.set_mark_list(get_menu_items())
-end
-
```

Comment:
`toggle_quick_menu` calls `on_menu_change` which would directly call this `save_changes`. Due to commit d0f4ad, there is now no need for this abstraction and `Marked.set_mark_list(get_menu_items())` has been moved to `on_menu_change`.